static/src/js/util/collections.js

```diff
@@ -27,12 +27,13 @@ export const withAdvancedSearch = (collectionParams, advancedSearch) => {
 
   // If we have a spatial value for the selectedRegion, use that for the spatial
   if (!isEmpty(selectedRegion) && selectedRegion.spatial) {
+    // Query spatial is saved as an array, but the selectedRegion spatial is not
     const { type } = selectedRegion
 
     if (type === 'reach') {
-      mergedParams.line = selectedRegion.spatial
+      mergedParams.line = [selectedRegion.spatial]
```

Comment:
May want a util that did this repeated work. Or at least a comment mentioning that were doing this in more than one place.